platforms/src/index.ts

```diff
@@ -0,0 +1,13 @@
+// Twitter Platform
+export { TwitterPlatform } from "./Twitter/App-Bindings";
+export { default as TwitterAuthProvider } from "./Twitter/Providers/TwitterAuthProvider";
+export {
+  TwitterFollowerGT100Provider,
+  TwitterFollowerGT500Provider,
+  TwitterFollowerGTE1000Provider,
+  TwitterFollowerGT5000Provider,
+} from "./Twitter/Providers/TwitterFollowerProvider";
+export { TwitterTweetGT10Provider } from "./Twitter/Providers/TwitterTweetsProvider";
+export { TwitterPlatformDetails, TwitterProviderConfig } from "./Twitter/Providers-config";
+
+export * as Ens from "./Ens";
```

Comment:
You should pull in the lateset changes from main. This file was removed. There is another export one directory up